cmake/public/LoadHIP.cmake

```diff
@@ -138,6 +145,26 @@ find_package_and_print_version(HIP 1.0)
 if(HIP_FOUND)
   set(PYTORCH_FOUND_HIP TRUE)
 
+  # Find ROCM version for checks
+  file(READ "${ROCM_PATH}/.info/version-dev" ROCM_VERSION_DEV_RAW)
+  string(REGEX MATCH "^([0-9]+)\.([0-9]+)\.([0-9]+)-.*$" ROCM_VERSION_DEV_MATCH ${ROCM_VERSION_DEV_RAW})
+  if(ROCM_VERSION_DEV_MATCH)
+    set(ROCM_VERSION_DEV_MAJOR ${CMAKE_MATCH_1})
+    set(ROCM_VERSION_DEV_MINOR ${CMAKE_MATCH_2})
+    set(ROCM_VERSION_DEV_PATCH ${CMAKE_MATCH_3})
+    set(ROCM_VERSION_DEV "${ROCM_VERSION_DEV_MAJOR}.${ROCM_VERSION_DEV_MINOR}.${ROCM_VERSION_DEV_PATCH}")
+  endif()
+  if(ROCM_VERSION_DEV VERSION_GREATER_EQUAL "4.1.0")
+    message("version is >= 4.1.0")
+  else()
+    message("version is < 4.1.0")
+  endif()
```

Comment:
we don't need this part any more.  it was just for testing purposes, to make sure the above parsing worked correctly.  the print statements below are good enough to verify parsing worked ok.

Reply:
Done